cmd/main.go

```diff
@@ -112,6 +112,7 @@ func main() {
 
 	rootCmd.AddCommand(
 		newInitProjectCmd(),
+		newMultiGroupProjectCmd(),
```

Comment:
The new cmd to enable the multigroup layout.

Reply:
I'm wondering what'd be the best layout for a new command.
I'm think about `kubebuilder edit --multigroup=<boolean>`
`kubebuilder edit` may be useful in the future e.g. enabling webhook kustomize patches
@DirectXMan12 @droot WDYT?

Reply:
I will leave this decision for you folks. Just let me know. 
However:
- Thinking in usability I'd prefer `kb enable --<option>` or `kb --<option>=<bool>` ( true by default ) as it was done. 
- Thinking in maintenance ability I would keep `kb --<option>=<bool>` .

Reply:
I really dislike the `kubebuilder --multigroup[=<bool>]` because you are adding functionality to the base kubebuilder command when everything else is under subcommands. Subcommands act like verbs in sentences and make the instruction easy to remember to user (UX is quite important in CLI design).

I would opt for one of the following options:
1- `kubebuilder edit --multigroup=<bool>`: in this case I consider the bool mandatory as the subcommand is not descriptive enough any other way.
2- `kubebuilder enable multigroup`: in this case, I would make `multigroup` a parameter and not a flag. Also, equivalent `kubebuilder disable multigroup` and even a  `kuebebuilder toggle multigrup` would also need to be added.